apps/user-portal/src/models/form-wrapper.ts

```diff
@@ -170,19 +170,14 @@
     value?: string;
 }
 
-/**
- * Group style types
- */
- export type GroupStyle = "grouped" | "inline";
-
 /**
  * Group model
  */
  export interface Group {
     startIndex: number;
     endIndex: number;
-    style: GroupStyle;
-    width?: SemanticWIDTHS | "equal";
+    wrapper: React.ComponentType;
+    wrapperProps: any;
```

Comment:
Can we avoid `any` here ?

Reply:
Here we can wrap a group of elements using a React component specified using the object attribute wrapper. wrapperProps is used to specify the props of the aforementioned React Component. Since different React Components take different prop values, I don't think we can have a specific type for this attribute?

Reply:
Cool, no worries then.